nir/src/main/scala/scala/scalanative/nir/Shows.scala

```diff
@@ -120,6 +120,11 @@ object Shows {
     case Op.Select(cond, thenv, elsev) =>
       sh"select $cond, $thenv, $elsev"
 
+    case Op.Box(code, obj) =>
+      sh"box $code $obj"
+    case Op.Unbox(code, obj) =>
+      sh"unbox $code $obj"
```

Comment:
1. Move them below to the high-level group.
2. `box[$ty] $value` and `unbox[$ty] $obj`